site/docs/main/troubleshooting.md

```diff
@@ -84,6 +84,12 @@ Replace `contour cli lds` with `contour cli rds` for RDS, `contour cli cds` for
 
 See [the deployment documentation][5] for some tips on using these two deployment options successfully.
 
+## Enabling Contour debug logging
+
+The `contour serve` subcommand has two command-line flags that can be helpful for debugging.
+The `--debug` flag enabled general Contour debug logging, which logs more information about how Contour is processing API resources.
+The `--kubernetes-debug` flag enabled verbose logging in the Kuberenetes client API, which can help debug interactions between Contour and the Kubernetes API server.
```

Comment:
Need to add to this example that you are required to pass a log level. Should also put some notes as to what log levels you might set (e.g. 10 vs 2).

Reply:
The levels are whatever client-go does, but I added some text to say that higher numbers give more logs :)